cypress/e2e/po/pages/global-settings/banners.po.ts

```diff
@@ -0,0 +1,119 @@
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import CheckboxInputPo from '@/cypress/e2e/po/components/checkbox-input.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import ColorInputPo from '@/cypress/e2e/po/components/color-input.po';
+import RootClusterPage from '@/cypress/e2e/po/pages/root-cluster-page';
+import RadioGroupInputPo from '@/cypress/e2e/po/components/radio-group-input.po';
+
+export class BannersPagePo extends RootClusterPage {
+  static url = '/c/_/settings/banners';
+  static goTo(): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(BannersPagePo.url);
+  }
+
+  constructor() {
+    super(BannersPagePo.url);
+  }
+
+  // input
+  headerInput(): LabeledInputPo {
+    return new LabeledInputPo(cy.getId('text-area-auto-grow').eq(0));
```

Comment:
I think i'm derping, isn't `eq` equals, or does it select the first instance of an array of cypress elements?

Reply:
its the later. it grabs a dom element at a specific index in an array of elements. in this case, we're targeting the first of three elements (with id `text-area-auto-grow`) found on the page